addons/escoria-core/game/core-scripts/esc_exit.gd

```diff
@@ -3,42 +3,54 @@ extends ESCItem
 class_name ESCExit, "res://addons/escoria-core/design/esc_exit.svg"
 
 
+# Name to use for calling this event.
+const EVENT_EXIT_SCENE = "exit_scene"
+
+
 # Path to the target scene to change to
 export(String, FILE, "*.tscn") var target_scene = ""
 
 # Sound effect to play when changing the scene
 export(String, FILE, "*.ogg,*.mp3,*.wav") var switch_sound = ""
 
+# ESC commands kept around for references to their command names.
+var _play_snd: PlaySndCommand
+var _change_scene: ChangeSceneCommand
+
 
 func _enter_tree():
 	is_exit = true
 	player_orients_on_arrival = false
-	default_action = "walk"
 
 
 func _ready():
+	_play_snd = PlaySndCommand.new()
```

Comment:
I didn't understand what you meant why you needed the command names when talking about it on Discord, but this is pretty cool. Very good idea!